backend/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM gradle:7.4.1-jdk11-alpine AS build
+COPY --chown=gradle:gradle . /home/gradle/src
+WORKDIR /home/gradle/src
+RUN gradle build --no-daemon
+
+FROM openjdk:11-jre-slim
+
+EXPOSE 8080
+
+RUN mkdir /app
+
+COPY --from=build /home/gradle/src/build/libs/*.jar /app/backend.jar
```

Comment:
This line does not work. Due to wildcard it is treated as copying multiple files to one file. We have to put artifact filename statically or resolve it other way.